src/components/atoms/Typography/Typography.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import * as I from '../../../models/typography/typography';
+import * as E from '../../../containers/enums/typography/typography';
+import * as S from './style';
+
+const Typography: React.FC<I.Typography> = ({ text, size, weight = E.weight.BOLD, color }) => {
+  return (
+    <S.Typography color={color} size={size} weight={weight}>
```

Comment:
didn't use material-ui?

Reply:
No. I decided use styled-component to this atom.